src/pages/pt-BR/core-concepts/project-structure.md

```diff
@@ -0,0 +1,107 @@
+
+Alguns arquivos (como componentes Astro) nem sequer são enviados ao navegador como foram escritos, mas sim são renderizados como HTML estático. Outros arquivos (como CSS) são enviados ao navegador mas podem ser otimizados e empacotados com outros arquivos CSS para melhorar a performance.
+
+### `src/components`
+
+**Componentes** são pedaços reutilizáveis de código para suas páginas HTML. Eles podem ser [componentes Astro](/pt-BR/core-concepts/astro-components) ou [componentes Frontend](/pt-BR/core-concepts/framework-components) como React ou Vue. É comum agrupar e organizar todos os componentes do seu projeto nesta pasta.
+
+Essa é uma prática comum em projetos Astro, porém não é obrigatória. Sinta-se livre para organizar seus componentes como desejar!
+
+### `src/layouts`
+
+[Layouts](/pt-BR/core-concepts/layouts) são um tipo especial de componente que envolvem algum conteúdo como parte de uma página maior. São mais frequentemente utilizados por [páginas Astro](/pt-BR/core-concepts/astro-pages) e [páginas Markdown](/pt-BR/guides/markdown-content) para definir o layout dessas páginas.
+
+Assim como o diretório `src/components`, é uma prática comum porém não obrigatória.
+
+### `src/pages`
+
+[Páginas](/pt-BR/core-concepts/astro-pages) são um tipo especial de componente utilizado para criar novas páginas no seu site. Uma página pode ser um componente Astro ou um arquivo Markdown que representa o conteúdo de uma página do seu site.
+
+> ⚠️  `src/pages` é um sub-diretório **obrigatório** em seu projeto Astro. Sem ele, seu site não terá páginas ou rotas!
+
+### `src/styles`
+
+É uma prática comum guardar seus arquivos CSS ou Sass no diretório `src/styles` porém não é obrigatório. Enquanto os seus estilos estiverem em algum lugar do diretório `src/` e forem importados corretamente, Astro irá manipular e otimizar esses arquivos.
+
+### `public/`
+
+O diretório `public/` é para arquivos e assets que não precisam ser processados durante o processo de build do Astro. Esses arquivos serão copiados para a pasta de build intocados.
+
+Esse comportamento faz com que `public/` seja ideal para assets comuns como imagens e fontes, ou arquivos especiais como `robots.txt` e `manifest.webmanifest`.
+
+Você pode colocar CSS e JavaScript no seu diretório `public/`, porém esteja atento que estes arquivos não serão empacotados ou otimizados na sua build final.
+
+ 💡 *Como uma regra geral, qualquer CSS ou JavaScript que você mesmo escrever deve estar no seu diretório `src/`.*
+
+### `package.json`
+
+Este é um arquivo utilizado por gerenciadores de pacotes de JavaScript para gerenciar suas dependências. Ele também define scripts que são comumente usados para rodar o Astro (ex: `npm start`, `npm run build`).
+
+Para ajuda em como criar um novo arquivo `package.json` para o seu projeto, veja as instruções de [instalação manual](/pt-BR/install/manual).
```

Comment:
little typo, I think
```suggestion
Para ajudar em como criar um novo arquivo `package.json` para o seu projeto, veja as instruções de [instalação manual](/pt-BR/install/manual).
```

Reply:
It was not technically a typo, but I think we should change the wording a bit to be clear about it:

```suggestion
Caso precise de ajuda em como criar um novo arquivo `package.json` para o seu projeto, veja as instruções de [instalação manual](/pt-BR/install/manual).
```